packages/nextjs/app/page.tsx

```diff
@@ -24,11 +33,17 @@ const Home: NextPage = () => {
     setStorage(new Bee("http://92.63.194.135:3000"));
   });
 
-  const data = projectlist
+  const all_open_projects = projectlist
```

Comment:
Let's move this 2 variables inside UserWorker and UserEmployer components. Now we calculate it in any case even if we don't go to the Employer or Worker tab, processing only on appropriate tab is more efficient.